.github/workflows/publish-results.yml

```diff
@@ -124,11 +125,11 @@ jobs:
                 }
 
                 // Move plots
-                if (fs.existsSync(plotsDir)) {
```

Comment:
merging since this was incorrectly merged to `master` and prevents the publishing job from running